digitalfabrik/entitlementcard
# 1699: Card design koblenz

### Short description

Adjust the koblenz card design

### Proposed changes



- replace logos
- set theme color
- adjust card branding
- add extra upscale for header logo

Note: I adjusted the header right logo a bit because fine adjustments with the current card branding are difficult. At some point we gonna have to create separate card template widgets per project

<img src="https://github.com/user-attachments/assets/8795ca7d-45ea-47b2-ad9a-8b80567ecee8" width=35% height=20%>

<img src="https://github.com/user-attachments/assets/a3500426-25f5-4e73-ab6f-f5047155fc6e" width=35% height=20%>


### Side effects



- none

### Testing

1. Start koblenz local and activate card
2. Check the other apps if the card design doesn't break

### Resolved issues



Fixes: #1699 
Fixes: #1741
